### DISCUSSION TITLE
Is there a way to duplicate a project?

### DISCUSSION THREAD

**Discussion by SaraVieira** [supabase/supabase#1732] [Questions]
Hello 👋

I have a project with some users and a database but i would love to be able to login in localhost so I was wondering if there is a simple way to duplicate a project, including the data 

These two projects do not have to stay in sync afterwards and i know I'll need a new GitHub API key for login but it would be nice to allow local login and also to have a staging environment

**Comment by silentworks** [comment]
I don't think there is a way to duplicate a project at the moment, but you can log into the database using the settings found under **settings/database** of the project and export all the tables. I'm not sure how you would export all the users, but maybe you could export the auth schema too.
